src/features/auth/components/LogInButton/LogInButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { ButtonInsideText } from 'ui/components/buttons/buttonInsideText/ButtonInsideText'
+import { TouchableLink } from 'ui/components/touchableLink/TouchableLink'
+import { Connect } from 'ui/svg/icons/Connect'
+import { Spacer, Typo } from 'ui/theme'
+// eslint-disable-next-line no-restricted-imports
+import { ColorsEnum } from 'ui/theme/colors'
+
+interface Props {
```

Comment:
`FunctionComponent` est cool

par défaut, il ajoute implicitement un `children`

je pense qu'on ne veut pas etre autorisé à faire

```ts
<LogInButton><SomeChildren /></LogInButton>
```

donc dans ce cas, on en veut pas

je recommande d'ajouter

```suggestion
children?: never
```